tweet_sentiment.py

```diff
@@ -1,42 +1,53 @@
 import re
 
 def calculate_sentiment_score(tweet):
-	d = {}
-	fl = open("words.txt", 'r')
-	for line in fl:
-		word, score = line.split("\t")
-		d[word] = int(score)
-	fl.close()
-	lst = tweet.split(' ')
-	score = 0
-	i = 0
-	l = len(lst)
-	while i < l:
-		if i == l-1:
-			if lst[i] in d:
-				score += d[lst[i]]
-		elif i == l-2:
-			str2 = lst[i] + ' ' + lst[i+1]
-			if str2 in d:
-				score += d[str2]
-				i += 1
-			elif lst[i] in d:
-				score += d[lst[i]]
-		else:
-			str3 = lst[i] + ' ' + lst[i+1] + ' ' + lst[i+2]
-			str2 = lst[i] + ' ' + lst[i+1]
-			if str3 in d:
-				score += d[str3]
-				i += 2
-			elif str2 in d:
-				score += d[str2]
-				i += 1
-			elif lst[i] in d:
-				score += d[lst[i]]
-		i += 1
-	return score
+    d = {}
+    fl = open("words.txt", 'r')
+    for line in fl:
+        word, score = line.split("\t")
+        d[word] = int(score)
+    fl.close()
+    lst = tweet.split(' ')
+    score = 0
+    i = 0
+    nopos = 0
+    noneg = 0
+    l = len(lst)
+    while i < l:
+        if d[lst[i]] > 0:
+            nopos += 1
+        elif d[lst[i]] < 0:
+            noneg += 1
```

Comment:
This will give us the wrong number of negative and positive terms. `words.txt` contains some terms (having more than one word) also so we should count that instead of counting only a single word.